test/common/scorecard.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2021 The Operator-SDK Authors
```

Comment:
It shows addressed in another pr either. Then, requires to rebase with the master before we merge this one. 
See; https://github.com/operator-framework/operator-sdk/pull/4552